target/linux/ramips/dts/mt7620a_apexera_netbot.dts

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera,netbot", "ralink,mt7620a-soc";
+	model = "Apexera NetBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+	};
+
+	aliases {
+		led-boot = &led_wifi;
+		led-failsafe = &led_wifi;
```

Comment:
It is discouraged to hijack LEDs with other purposes here. If you only have LEDs which would be ambiguous on a running device, remove the aliases section.

Reply:
I have read the SCH of this device, And it has only one led that could be used as the alias function, and I think is necessary to tell the user which state it's working, so I choose this.

Reply:
Still, I do not like this ambiguity, please remove the aliases in order to move this forward.